src/test/resources/META-INF/services/org.junit.platform.launcher.TestExecutionListener

```diff
@@ -0,0 +1,2 @@
+org.broadinstitute.consent.http.db.DAOTestHelper
```

Comment:
Adding metadata to a text file seems bad but I guess there's no better way to do this since we need the code to run before the test suite is executed.